rpc/client.go

```diff
@@ -26,212 +26,606 @@ type Transport interface {
 	Close() error
 }
 
-type Client interface {
-	Logger() log.Logger
-	ServerConfig() ServerConfig
-	User() discord.User
-	V() int
-	Transport() Transport
-
-	Subscribe(event Event, args CmdArgs, handler Handler) error
-	Unsubscribe(event Event, args CmdArgs) error
-	Send(message Message, handler Handler) error
-	Close()
+type responseMessage struct {
+	data MessageData
+	err  error
 }
 
 func NewClient(clientID snowflake.ID, opts ...ConfigOpt) (Client, error) {
```

Comment:
```suggestion
func New(clientID snowflake.ID, opts ...ConfigOpt) (Client, error) {
```

Reply:
fixed in [4761252](https://github.com/dennisrijsdijk/disgo/commit/47612524627647d77f406eb356b1f85dfa952f1e)